android/src/androidTest/java/bbct/android/common/activity/test/BaseballCardDetailsAddCardsTest.java

```diff
@@ -47,10 +47,11 @@
     public SupportFragmentTestRule fragmentTestRule =
             new SupportFragmentTestRule(new BaseballCardDetails());
 
-    private static final String CARD_DATA = "three_cards.csv";
+    //  private static final String CARD_DATA = "three_cards.csv";
+    private static final String CARD_DATA = "cards_alphanumeric.csv";
 
     private List<BaseballCard> allCards = null;
-    private BaseballCard card = null;
+    private BaseballCard card = null, newCard = null;
```

Comment:
`newCard` should be declared on its own line and given a more descriptive name. You should consider declaring it as a local variable if it is used in only one test.